frontend/src/components/project/ProjectMetaData.js

```diff
@@ -99,15 +102,15 @@ const WithDescription = ({ className, project_parent, hovering, project, main_pr
             <PlaceIcon className={classes.cardIcon} />
           </Tooltip>
           <Typography className={classes.metadataText}>{project.location}</Typography>
-          <Collapse in={hovering} timeout={500}>
+          {/* Defer to MUI's best guess on height calculation for timeout: https://material-ui.com/api/collapse/ */}
+          <Collapse in={hovering} timeout="auto">
             <Typography className={classes.shortDescription}>
               {project.short_description}
             </Typography>
           </Collapse>
-          {!hovering && <Categories main_project_tag={main_project_tag} />}
         </Box>
       </Container>
-      {hovering && <Categories main_project_tag={main_project_tag} hovering={hovering} />}
+      <Categories main_project_tag={main_project_tag} hovering={true} />
```

Comment:
Why is hovering set to always true here? One problem this causes is that this always adds a background to the categories which looks a bit weird if the cards have different height (see screenshots)
![Screenshot from 2020-12-21 18-39-49](https://user-images.githubusercontent.com/10531079/102810318-cb188e80-43c3-11eb-9814-d80a3e2e840e.png)

Reply:
Good call! Backed this out. The intent was to minimize a minor layout shift when hovering, like here:


https://user-images.githubusercontent.com/1530684/104094569-89eb0000-5246-11eb-8f0b-154b022d58cf.mov